build-tools/packages/version-tools/src/bumpTypes.ts

```diff
@@ -86,21 +86,24 @@ export type InterdependencyRange =
  * excluding non-conforming strings than it does at finding valid ones. This appears to be sufficient for our needs,
  * which is good because I don't know how to fix it.
  */
-export function isInterdependencyRange(r: any): r is InterdependencyRange {
-	if (semver.valid(r) !== null) {
+export function isInterdependencyRange(r: unknown): r is InterdependencyRange {
+	if (semver.valid(r as string | semver.SemVer) !== null) {
 		return true;
 	}
 
-	if (RangeOperators.includes(r) || WorkspaceRanges.includes(r)) {
+	if (
+		RangeOperators.includes(r as RangeOperator) ||
+		WorkspaceRanges.includes(r as WorkspaceRange)
```

Comment:
These helpers are using `as` reasonably and could be used here:
```suggestion
		isRangeOperator(r) ||
		isWorkspaceRange(r)
```